---
title: Sample Projects
---

### Exploring splicing landscapes in single-cells with long-read sequencing
This is a project I concieved as part of a computational single-cell immunology course. Using long-read technology, I wanted to investigate whether or not gene and isoform expression hold the same biological information or if we could elucidate sub cell states and types from the isoform data that are not found in the gene expression data.

In colorectal cancer tumor cells [View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/crc_splicing.ipynb) <br />

In lung cancer cell cell lines [View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/lung_cell_line_splicing.ipynb)

### Introduction to machine learning notebook
Co-authored with Prakruthi Burra. We designed this notebook as part of a biannual Python bootcamp that we teach at UC Berkeley.

[View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/ml_bootcamp.ipynb)


### A simple autoencoder for single cell immune cells
[View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/sc_autoencoder.ipynb)

### Bagging classifier for single cell immune cells
[View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/sc_classification.ipynb)

### Data engineering and classification of different exoplanet discovery systems
[View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/exoplanet_project.ipynb)

### Univariate vs Multivariate Autoregressive modeling of Covid-19 spread
[View notebook →](https://nbviewer.org/urls/carmelle-cat.github.io/notebooks/modeling_covid_spread.ipynb)
